cmd/instance.go

```diff
@@ -90,7 +90,7 @@ func instanceCreateCommand() *cobra.Command {
 
 	cmdInstanceCreate.PersistentFlags().StringVarP(&config, "config", "c", "", "config for nanos")
 	cmdInstanceCreate.PersistentFlags().StringVarP(&imageName, "imagename", "i", "", "image name [required]")
-	cmdInstanceCreate.PersistentFlags().StringVarP(&flavor, "flavor", "f", "g1-small", "flavor name for GCP")
```

Comment:
again, can you set g1-small as the default in gcloud if not specified here please?

Reply:
this was resolved in a diff. pr https://github.com/nanovms/ops/pull/709/files#diff-7a003c5767841ecce9f30f3795aee8eb0b854a61fcf1b4efd04357d262ec883dR353